Doc/glossary.rst

```diff
@@ -1202,6 +1202,12 @@ Glossary
       :func:`sys.getrefcount` function to return the
       reference count for a particular object.
 
+      In :term:`CPython`, reference counts are not considered to be stable
+      or well-defined values; the number of references to an object, and how
+      that number is affected by Python code, may be different between
+      versions. Consequently, don't rely on an object's reference count to be
```

Comment:
The last line is not true anymore with free-threading and interpreter skipping ref on temporaries, I suggest to remove the last line.